docs/index.md

```diff
@@ -12,23 +12,23 @@ We define an autonomous economic agent or AEA as:
 
 <!-- Autonomous Economic Agents are digital entities that run complex dynamic decision-making algorithms for application owners and clients. -->
 
 AEAs are not:
 
-* just any agents: AEAs' purpose is to generate economic value in a multi-stakeholder environment with competing incentives between agents. The represent humans, organisations or things.
+* just any <a href="https://en.wikipedia.org/wiki/Software_agent"  target="_blank">agents</a>: AEAs' purpose is to generate economic value in a multi-stakeholder environment with competing incentives between agents. They represent humans, organisations or things.
 * APIs or sensors which do not have agency.
-* smart contracts which do not display any proactiveness and are purely reactive to external requests (=contract calls). 
-* artificial general intelligence (AGI): AEAs can have a very narrow goal directed focus involving some economic gain and implemented via simple conditional logic.
+* <a href="https://en.wikipedia.org/wiki/Smart_contract" target="_blank">smart contracts</a> which do not display any proactiveness and are purely reactive to external requests (=contract calls and transactions). 
+* <a href="https://en.wikipedia.org/wiki/Artificial_general_intelligence" target="_blank">artificial general intelligence (AGI)</a>: AEAs can have a very narrow goal directed focus involving some economic gain and implemented via simple conditional logic.
 
 
 ## What is the AEA Framework?
 
 The AEA framework is a Python-based development suite which equips you with an efficient and accessible set of tools for building AEAs. The framework is modular, extensible, and composable. It attempts to make agent development as straightforward an experience as possible, similar to web development using popular web frameworks.
 
-AEAs achieve their goals with the help of the <a href="oef-ledger">Open Economic Framework (OEF)</a> - a decentralized communication and search & discovery system for agents - and using <a href="oef-ledger">Fetch.ai's blockchain</a> as a financial settlement and commitment layer. Third-party blockchains, such as <a href="https://ethereum.org/en/" target="_blank">Ethereum</a>, may also allow AEA integration.
+AEAs achieve their goals with the help of a decentralized search & discovery system for agents the <a href="oef-ledger">simple Open Economic Framework (sOEF)</a>, the <a href="acn">Agent Communication Network</a> - a decentralized communication system for agents - and using <a href="oef-ledger">Fetch.ai's blockchain</a> as a financial settlement and commitment layer. Third-party blockchains, such as <a href="https://ethereum.org/en/" target="_blank">Ethereum</a>, may also allow AEA integration.
```

Comment:
```suggestion
AEAs achieve their goals with the help of a decentralized search & discovery system for agents the <a href="oef-ledger">simple Open Economic Framework (sOEF)</a>, the <a href="acn">Agent Communication Network</a> - a decentralized communication system for agents - and using <a href="oef-ledger">Fetch.ai's blockchain</a> as a financial settlement and commitment layer. AEA could also be integrated with third-party blockchains, such as <a href="https://ethereum.org/en/" target="_blank">Ethereum</a>.
```

Reply:
The `decentralized` adj. for search and discovery may be problematic when a link to sOEF follows it because the latter isn't decentralised.